content/en/developers/reference/json-rpc/eth/index.md

```diff
@@ -23,7 +23,7 @@ This method is intended to return a list of addresses owned by client. However,
 Example:
 
 ```curl
-curl --location --request POST 'https://wallaby.node.glif.io/rpc/v0' \
+curl --location --request POST 'https://api.hyperspace.node.glif.io/rpc/v0' \
```

Comment:
I am not a huge fan  that the docs should point in testnet by defualt - do we plan to change this back when fevm launch in mainnet? if so, can we open an issue to make sure we dont forget about it?